bin/gltf-pipeline.js

```diff
@@ -55,6 +64,8 @@ var options = {
     quantize : quantize,
     encodeNormals : encodeNormals,
     compressTextureCoordinates : compressTextureCoordinates,
+    aoOptions : aoOptions,
+    imageProcess : defined(aoOptions) && aoOptions.toTexture,
```

Comment:
I removed `imageProcess` in https://github.com/AnalyticalGraphicsInc/gltf-pipeline/pull/128, so you can delete this here.